jboss-eap-config-elytron/module.yaml

```diff
@@ -0,0 +1,26 @@
+schema_version: 1
+name: jboss.eap.config.elytron
+version: '1.0'
+description: EAP Elytron configuration scripts
+execute:
+- script: configure.sh
+  user: '185'
+envs:
+    - name: "CONFIGURE_ELYTRON_SSL"
+      example: "false"
```

Comment:
Should this be 'true', since the only reason to set the var is to make it true?

Reply:
Sure, that makes sense to me.